salt/modules/file.py

```diff
@@ -3179,8 +3179,9 @@ def link(src, path):
     try:
         os.link(src, path)
         return True
-    except (OSError, IOError):
-        raise CommandExecutionError('Could not create \'{0}\''.format(path))
+    except (OSError, IOError) as e:
```

Comment:
I know you said you still have some work to do here, but right off the bat, can you change these `e` references to something like `err` or `error`? We like to have 3-letter variables. See [here](https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#variables) for more information.